src/client/components/leaderboard/leaderboard.js

```diff
@@ -26,7 +36,10 @@ class Leaderboard extends React.Component {
               {this.props.scores.map((val,idx) => 
                 <tr key={idx}>
                   <td>{idx}</td>
-                  <td>{this.props.names[idx]} <strong>{(parseInt(this.props.playerID) === idx) ? '(you)' : ""}</strong></td>
+                  <td>{this.props.names[idx]} <strong> {
+                  (parseInt(this.props.playerID) === idx) ? '(you)' : ""}</strong></td>
+                  <td>{
+                   haspassed(idx) }</td>
```

Comment:
The problem with this approach is that the table changes its width when the first "passed" appears.